eng/Subsets.props

```
@@ -116,12 +102,12 @@
     <SubsetName Include="CoreHost" Description="The .NET Core hosts." />
     <SubsetName Include="Installer.NoCoreHost" Description="Hosting libraries, bundles, and installers. Includes these projects' tests." />
     <SubsetName Include="Installer.Managed" Description="The managed .NET hosting projects. This includes HostModel." />
-    <SubsetName Include="Installer.DepProjs" Description="The dependency projects. These gather shared framework files and run crossgen on them to turn them into ready-to-run (R2R) assemblies for the current platform." />
     <SubsetName Include="Installer.PkgProjs" Description="The packaging projects. These produce NETCoreApp assets: NuGet packages, installers, zips, and Linux packages." />
     <SubsetName Include="Bundles" Description="The shared framework bundle installer projects. Produces .exe installers for Windows." />
     <SubsetName Include="Installers" Description="Generates additional installers. This produces the shared frameworks and their installers." />
     <SubsetName Include="Installer.Tests" Description="The test projects. Note that building this doesn't execute tests: you must also pass the '-test' argument." />
     <SubsetName Include="Installer.Publish" OnDemand="true" Description="Builds the installer publish infrastructure." />
+    <SubsetName Include="sfx" OnDemand="true" Description="Builds the shared frameworks, bundles, installers with the new shared framework SDK." />
```

```suggestion
    <SubsetName Include="sfx" OnDemand="true" Description="Builds the shared frameworks, bundles, installers." />
```
It is not interesting for consumers that it uses the new shared framework SDK.